applications/DEMApplication/custom_constitutive/DEM_KDEM_with_damage_CL.cpp

```diff
@@ -99,7 +99,9 @@ namespace Kratos {
                 sliding,
                 r_process_info);
 
-        FindMaximumValueOfNormalAndTangentialDamageComponents();
+        // FindMaximumValueOfNormalAndTangentialDamageComponents();
+        
+        CalculationOfNormalAndTangentialDamageComponents();
```

Comment:
Name of functions must a an imperative verb. `CalculateNormalAndTangentialDamageComponents`

Reply:
Modified. Thanks!